en/identity-server/6.0.0/docs/deploy/security/enabling-mutual-ssl.md

```diff
@@ -0,0 +1,96 @@
+# Enabling Mutual SSL
+
+### How it works
+
+In contrast to the usual one-way SSL authentication where a client
+verifies the identity of the server, in mutual SSL the server validates
+the identity of the client so that both parties trust each other. This
+builds a system that has a very tight security and avoids any requests
+made to the client to provide the username/password, as long as the
+server is aware of the certificates that belong to the client.
+
+Before the process begins, the client and servers certificates are stored
+in there relevant `          keystores         ` . In the case of JAVA
+they are `          jks         ` files. Let's take a look at where the
+JKS files are saved:
+
+-   WSO2 Identity Server certificates are stored in the
+    `           <IS-HOME>/repository/resources/security/wso2carbon.jks          ` file.
+-   Server side certificates are stored in the
+    `           <IS-HOME>/repository/resources/security/clienttruststore.jks          ` file.
+
+These certificates are signed and issued by a certificate authority that
+allows both the client and server to communicate freely. Now let's look
+at how it works:
+
+![Certificate exchange flow](../../assets/img/deploy/security/certificate-exchange-flow.jpg) 
+
+1.  The Client attempts to access a protected resource and the SSL/TSL
+    handshake process begins.
+2.  The Server presents its certificate, which is the
+    `           server.crt          ` according to our example as shown
```

Comment:
```suggestion
    `server.crt` according to our example as shown
```

Reply:
https://github.com/wso2/docs-is/pull/5163/commits/2142c9d4fbfecd4990316510453378f22463be7d